web-ui/src/components/personnel/Personnel.jsx

```diff
@@ -102,15 +160,46 @@ const createFeedbackRequestLink = (memberId) => (
             }}
           />
         </ListItemAvatar>
-        <ListItemText primary={name} secondary={createFeedbackRequestLink(person.id)}/>
+        <ListItemText primary={name} />
       </ListItem>
     );
   }
 
   // Create the entries for the personnel container
   const createPersonnelEntries = () => {
     if (personnel && personnel.length > 0) {
-      return personnel.map((person) => createEntry(person, selectMostRecentCheckin(state, person.id), null));
+      return personnel.map((person) =>
+        personnelEntries(
+          person,
+          selectMostRecentCheckin(state, person.id),
+          null
+        )
+      );
+    } else {
+      return [];
+    }
+  };
+
+  // Create entries for open checkins for former personnel
+  const createFormerPersonnelEntries = () => {
+    if (pastCheckins && pastCheckins.length > 0) {
+      const personnelIds = personnel.map((person) => person.id);
+      const pastPersonnelIds = pastCheckins
+        .filter((checkins) => !personnelIds.includes(checkins.teamMemberId))
+        .reduce((pastIds, checkins) => {
+          if (!personnelIds.includes(checkins.teamMemberId)) {
+            pastIds.push(checkins.teamMemberId);
+          }
+          return pastIds;
+        }, []);
+      return pastPersonnelIds.map((memberId) => {
+        const person = selectProfile(state, memberId);
+        return formerPersonnelEntries(
+          person,
+          selectMostRecentCheckin(state, memberId),
```

Comment:
I think we actually need a new selector here. This one returns the most recent check in...but we really want the most recent check in that the current user was PDL for.  So, I think I would create one that does something like:

selectMostRecentCheckinWithPDL(state, memberId, pdlId)